src/lib/ContractInfo.ts

```diff
@@ -27,6 +39,19 @@ export class VestingContractInfo {
         );
     }
 
+    public static objectToContractType(o: VestingContractInfoEntry): VestingContract {
+        return {
+            type: o.type,
```

Comment:
For `HashedTimeLockedContractInfo` this value is set explicitly in the corresponding function.